src/index.js

```diff
@@ -19,6 +19,10 @@ const admins = [
   'U032DGC20DS', // jenni s
   'U063ECDB83C', // jenni v
   'U062QJKGL72', // kirsi k
+  'U04Q49ZH1EW', // jouni Witted
```

Comment:
Maybe this whole list should be externalized?

Reply:
True, would be more maintainable. But given that these are not changing and most likely go away if other methods of invocation and authenticate is introduced.

Reply:
I'd think more in a view that this is public repo. So if some other would like to utilize this, now the "customizations" are kind of hidden inside the code.

Reply:
Moved list to be fetched from secrets instead of hardcoded list, added mention to readme.